cmd/kpod/create.go

```diff
@@ -230,6 +233,28 @@ func parseSecurityOpt(config *createConfig, securityOpts []string) error {
 		err       error
 	)
 
+	if config.pidMode.IsHost() {
+		labelOpts = append(labelOpts, label.DisableSecOpt()...)
+	}
+	if config.pidMode.IsContainer() {
+		ctr, err := config.runtime.LookupContainer(config.pidMode.Container())
+		if err != nil {
+			return errors.Wrapf(err, "container %q not found", config.pidMode.Container())
+		}
+		labelOpts = append(labelOpts, label.DupSecOpt(ctr.ProcessLabel())...)
+	}
+
+	if config.ipcMode.IsHost() {
+		labelOpts = append(labelOpts, label.DisableSecOpt()...)
+	}
+	if config.ipcMode.IsContainer() {
```

Comment:
Can this be else-if? I think host and container are exclusive

Reply:
Fixed